cmd/state/internal/cmdtree/cmdtree.go

```diff
@@ -215,6 +217,7 @@ func New(prime *primer.Values, args ...string) *CmdTree {
 		newCommitCommand(prime),
 		newPublish(prime),
 		newEvalCommand(prime),
+		manifestCmd,
```

Comment:
```suggestion
		newManifestCommmand(prime),
```

No need for a variable if you're only passing it here.